concourse/steps/scan_container_images.py

```diff
@@ -227,6 +227,49 @@ def result_to_tuple(upload_result: UploadResult):
     return table
 
 
+def create_filter(
```

Comment:
the name does not match what the function does (also, the function does a bit too much for my taste)